HomeEdu/Data/Model/ViewScore/ScoreData.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ScoreData.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 8/19/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+final class ScoreData: Mappable {
+    var id: String?
+    var studentId: Double?
+    var totalCredit: Int?
+    var totalGpa: Double?
+    var terms: [ScoreTerm]?
```

Comment:
```suggestion
    var terms = [ScoreTerm]()
```
array thì khởi tạo mảng rỗng luôn, tương tự với các class khác nhé